### DISCUSSION TITLE
Icons too detailed in v2.0.18

### DISCUSSION THREAD

**Discussion by davidfranke-dev** [tailwindlabs/heroicons#1040] [Ideas]
I have used v1 for a while, when the new update with the v2 icons came out. After switching all icons in my current project, however, I encountered that the new ones were often worse than the old ones. One might argue that this is a question of taste and I certainly agree to some degree. Taking into account, however, that in modern website design better often means more minimalistic the icons are quite regressive. One of the most striking examples is the Presentation Chart Bar Icon. 

New one:
![Screenshot 2023-07-19 125759](https://github.com/tailwindlabs/heroicons/assets/114422035/2af871e4-c9d3-4e1a-b7bb-a12fa814a163)

Old one:
![Screenshot 2023-07-19 125844](https://github.com/tailwindlabs/heroicons/assets/114422035/871e8b8e-c02d-40f6-b127-396a3cad5cdd)

The minimalistic style of the old one in my opinion just way better suits the current overall design situation. But what do you think? You are very welcome to share your opinion on the topic.

**Comment by quick007** [comment]
I personally love the more detailed designs. It means they can be scaled up a little bit more, and add that little extra bit of flair to a website. Minimalistic design seems to be changing from the militaristic "less is more" approach, and shifting toward a design scheme that still prioritizes a small amount of content/detail without getting in the way of a good design aesthetic. That's just my 2 cents though, a huge part of design is personal taste, so take my opinions however you like.
